specification/quota/resource-manager/Microsoft.Quota/preview/2019-07-19-preview/quota.json

```
@@ -0,0 +1,595 @@
+        "operationId": "QuotaRequests_Delete",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ProviderIdParameter"
+          },
+          {
+            "$ref": "#/parameters/LocationIdParameter"
+          },
+          {
+            "$ref": "#/parameters/QuotaRequestNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok. Quota Request deleted."
+          },
+          "404": {
+            "description": "Error: Not found. Quota request for provided quotaRequest name not found"
+          },
+          "409": {
+            "description": "Error: Conflict. Quota request for provided quotaRequest is in processing state. Request cannot be deleted."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ExceptionResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Quota/providers/{providerId}/locations/{locationId}/quotaRequests": {
+      "get": {
+        "tags": [
+          "QuotaRequests"
+        ],
+        "summary": "For the specified location and resource Provider, gets the current quota requests under the subscription over the time\n period of one year ago from now to one year back.",
+        "description": "For the specified location and Resource provider gets the current quota requests under the subscription over the time period of one year ago from now to one year back. oData filter can be used to select quota requests.",
+        "operationId": "QuotaRequest_status",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ProviderIdParameter"
+          },
+          {
+            "$ref": "#/parameters/LocationIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok. Sending Quota request status and details",
+            "schema": {
+              "$ref": "#/definitions/QuotaRequestDetailsByNameList"
+            }
+          },
+          "404": {
+            "description": "Error: Not found. Quota requests for provided quotaRequestId not found"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ExceptionResponse"
+            }
+          },
+          "x-ms-pageable": {
+            "nextLinkName": "nextpage"
```

If the camelCase doesn't make sense, you can add the word to https://github.com/Azure/azure-rest-api-specs/blob/master/custom-words.txt to fix the spellcheck failure
```suggestion
            "nextLinkName": "nextPage"
```